x-pack/plugins/security_solution/public/common/components/endpoint/page_view.tsx

```diff
@@ -25,15 +25,17 @@ import { EuiTabProps } from '@elastic/eui/src/components/tabs/tab';
 
 const StyledEuiPage = styled(EuiPage)`
   &.endpoint--isListView {
-    padding: 0 ${(props) => props.theme.eui.euiSizeL};
+    padding: 0;
+    flex: 1 1 100%;
```

Comment:
I'm not sure if I want to use **flex** directly, I'd rather find a way to use EUI classes but not sure it's as easy.